util/disk-image-validator/run_config_tests.py

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2025 The Regents of the University of California
+# All rights reserved.
```

Comment:
You should add a shebang here so you can execute this file directly

#!/usr/bin/env python3